addons/l10n_ro_edi_stock/models/l10n_ro_edi_stock_etransport_interface.py

```diff
@@ -0,0 +1,670 @@
+    ('252901', "BVF Orşova (ROCR7280)"),
+    ('252902', "BVF Porţile De Fier I (ROCR7270)"),
+    ('252903', "BVF Porţile De Fier II (ROCR7200)"),
+    ('252904', "BVF Drobeta Turnu Severin (ROCR9000)"),
+    ('262801', "BVI Târgu Mureş (ROBV8800)"),
+    ('262901', "BVF Târgu Mureş Aeroport (ROBV8820)"),
+    ('272801', "BVI Piatra Neamţ (ROIS7400)"),
+    ('282801', "BVI Corabia (ROCR2000)"),
+    ('282802', "BVI Olt (ROCR8210)"),
+    ('292801', "BVI Ploiești (ROBU7100)"),
+    ('302801', "BVI Satu-Mare (ROCJ7810)"),
+    ('302901', "BVF Halmeu (ROCJ4310)"),
+    ('302902', "BVF Aeroport Satu Mare (ROCJ7830)"),
+    ('312801', "BVI Zalău (ROCJ9700)"),
+    ('322801', "BVI Sibiu (ROBV7900)"),
+    ('322901', "BVF Sibiu Aeroport (ROBV7910)"),
+    ('332801', "BVI Suceava (ROIS8230)"),
+    ('332901', "BVF Dorneşti (ROIS2700)"),
+    ('332902', "BVF Siret (ROIS8200)"),
+    ('332903', "BVF Suceava Aero (ROIS8250)"),
+    ('332904', "BVF Vicovu De Sus (ROIS9620)"),
+    ('342801', "BVI Alexandria (ROCR0310)"),
+    ('342901', "BVF Turnu Măgurele (ROCR9100)"),
+    ('342902', "BVF Zimnicea (ROCR5800)"),
+    ('352802', "BVI Timişoara Bază (ROTM8720)"),
+    ('352901', "BVF Jimbolia (ROTM5010)"),
+    ('352902', "BVF Moraviţa (ROTM5510)"),
+    ('352903', "BVF Timişoara Aeroport (ROTM8730)"),
+    ('362901', "BVF Sulina (ROCT8300)"),
+    ('362902', "BVF Aeroport Delta Dunării Tulcea (ROGL8910)"),
+    ('362903', "BVF Tulcea (ROGL8900)"),
+    ('362904', "BVF Isaccea (ROGL8920)"),
+    ('372801', "BVI Vaslui (ROIS9610)"),
+    ('372901', "BVF Fălciu (-)"),
+    ('372902', "BVF Albiţa (ROIS0100)"),
+    ('382801', "BVI Râmnicu Vâlcea (ROCR7700)"),
+    ('392801', "BVI Focșani (ROGL3600)"),
+    ('402801', "BVI Bucureşti Poştă (ROBU1380)"),
+    ('402802', "BVI Târguri și Expoziții (ROBU1400)"),
+    ('402901', "BVF Băneasa (ROBU1040)"),
+    ('512801', "BVI Călăraşi (ROCT1710)"),
+    ('522801', "BVI Giurgiu (ROBU3910)"),
+    ('522901', "BVF Zona Liberă Giurgiu (ROBU3980)"),
+]
+
+STATE_CODES = {
+    'AB': '1',
+    'AR': '2',
+    'AG': '3',
+    'BC': '4',
+    'BH': '5',
+    'BN': '6',
+    'BT': '7',
+    'BV': '8',
+    'BR': '9',
+    'BZ': '10',
+    'CS': '11',
+    'CJ': '12',
+    'CT': '13',
+    'CV': '14',
+    'DB': '15',
+    'DJ': '16',
+    'GL': '17',
+    'GJ': '18',
+    'HR': '19',
+    'HD': '20',
+    'IL': '21',
+    'IS': '22',
+    'IF': '23',
+    'MM': '24',
+    'MH': '25',
+    'MS': '26',
+    'NT': '27',
+    'OT': '28',
+    'PH': '29',
+    'SM': '30',
+    'SJ': '31',
+    'SB': '32',
+    'SV': '33',
+    'TR': '34',
+    'TM': '35',
+    'TL': '36',
+    'VS': '37',
+    'VL': '38',
+    'VN': '39',
+    'B': '40',
+    'CL': '51',
+    'GR': '52',
+}
+
+
+class L10nRoEdiStockETransportInterface(models.Model):
+    _name = 'l10n_ro.edi.stock.etransport.interface'
+    _description = "Interface for Romanian eTransport"
+
+    picking_id = fields.Many2one(comodel_name='stock.picking', default=None)
```

Comment:
Why don't you put the EDI on the picking directly instead of creating a new model?